tests/requirements.txt

```diff
@@ -28,7 +28,7 @@ antsibull-changelog==0.30.0
     # via antsibull-docs
 antsibull-core==3.1.0
```

Comment:
You also need to use the `main` branch of antsibull-core.

Reply:
Ah